state/allwatcher.go

```diff
@@ -263,7 +263,7 @@ func (m *backingMachine) updated(st *State, store *multiwatcherStore, id string)
 		}
 
 		spaceID := addr.SpaceID
-		if spaceID != network.DefaultSpaceId && spaceID != "" {
+		if spaceID != network.AlphaSpaceId && spaceID != "" {
```

Comment:
Is it possible for the space ID to be empty after the migrations?

Reply:
apparently yes, for lxd:
ERROR juju.worker.modelcache worker.go:309 watcher error: space id "" not found, getting new watcher.  

Part of final move to new spaces paradigm will have to include revisiting all these checks.  :-)